### ISSUE TITLE
Empty repo!

### ISSUE THREAD

**Issue by mohsenuss91** [cherifGsoul/yii2-clean-architecture-docs#1]
@cherifGsoul  This repository is empty :(

**Comment by cherifGsoul** [comment]
Yeah sorry is an old one and forget about it, I'll do an example once I have more time.

**Comment by mohsenuss91** [comment]
> Yeah sorry is an old one and forget about it, I'll do an example once I have more time.

Waiting for it 👍🏻

**Comment by mgrechanik** [comment]
Wonder, why you keep this repo.
Nobody yet figured out how to create a clean code with this framework
